cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -73,7 +80,12 @@ DrawableSubmap::DrawableSubmap(const int trajectory_id, const int submap_index,
   material_->setCullingMode(Ogre::CULL_NONE);
   material_->setDepthBias(-1.f, 0.f);
   material_->setDepthWriteEnabled(false);
-  scene_node_->attachObject(manual_object_);
+  visibility_ = ::cartographer::common::make_unique<::rviz::BoolProperty>(
```

Comment:
no need for a unique_ptr here IIUC. make it a value member in the class and initialize in the initializer list.